src/machine/machine-varlink.c

```diff
@@ -188,3 +188,68 @@ int vl_method_register(sd_varlink *link, sd_json_variant *parameters, sd_varlink
 
         return sd_varlink_reply(link, NULL);
 }
+
+int lookup_machine_by_name(sd_varlink *link, Manager *manager, const char *machine_name, Machine **ret_machine) {
```

Comment:
The function is split out in the first commit, then moved in the third commit. Please do that in the first commit.